packages/tasks/src/local-apps.ts

```diff
@@ -214,6 +244,15 @@ export const LOCAL_APPS = {
 			(model.pipeline_tag === "text-generation" || model.pipeline_tag === "image-text-to-text"),
 		snippet: snippetVllm,
 	},
+	tgi: {
+		prettyLabel: "TGI",
+		docsUrl: "https://huggingface.co/docs/text-generation-inference/",
+		mainTask: "text-generation",
+		displayOnModelPage: (model: ModelData) =>
+			(isAwqModel(model) || isGptqModel(model) || isMarlinModel(model) || isTransformersModel(model)) &&
+			isTgiModel(model),
```

Comment:
Should this only be in the case `isTgi`? Otherwise it seems we have two different heuristic sets for deciding whether a model is TGI-compatible, one for the tag and one for the display

Reply:
Good point, however, we can load all AWQ + GPTQ models in TGI as well.

Reply:
Let me see if the tag already covers these use-cases.

Reply:
fixed in https://github.com/huggingface/huggingface.js/pull/962/commits/933d60bd5a3a6c7b342c8934e5edadf5e381ac00